shaders/mesh_shader_culling/mesh_shader_culling.mesh

```diff
@@ -51,7 +51,8 @@ void main()
     vec4 displacement = vec4(deltaPosition_xy, 0.0f, 1.0f);
 
     // the actual position of each meshlet:
-    vec4 position = displacement + sharedData.positionTransformation;
+    vec4 posTrans = vec4(sharedData.positionTransformation.x, sharedData.positionTransformation.y, 0.0, 0.0);
+    vec4 position = displacement + posTrans;
     float squareRadius = position.x * position.x + position.y * position.y;
```

Comment:
It's a bit surprising, that you extend `deltaPosition_xy` and `sharedData.positionTransform` from `vec2` to `vec4`, and then just use `x` and `y`.
Why not just
`vec2 position = deltaPosition_xy + sharedData.positionTransformation;`

Reply:
:) Did it without thinking.  Good catch